project/src/js/ui/tournament/detail/tournamentDetailView.js

```diff
@@ -200,3 +223,59 @@ function renderFooter(ctrl) {
   );
 }
 
+function tournamentPodium(podium) {
+  return (
+    <div className="podium">
+      { renderPlace(podium[1]) }
+      { renderPlace(podium[0]) }
+      { renderPlace(podium[2]) }
+    </div>
+  );
+}
+
+function renderPlace(data) {
+  const rank = data.rank;
+  return (
+    <div className={'place'+rank}>
+      <div className="trophy"> </div>
+      <div className="username" config={helper.ontouch(() => m.route('/@/' + data.name))}>
+        {data.name}
+      </div>
+      <div className="rating"> {data.rating} {helper.progress(data.ratingDiff)} </div>
+      <table className="stats">
```

Comment:
I'd rather not have a table here. It's really easy to do in flex, I can do it if you want.